.github/workflows/import-strings.yml

```diff
@@ -34,6 +34,7 @@ jobs:
     - name: Update new strings
       run: |-
         git diff || (git add Blockzilla/*/*.lproj/* Blockzilla/*.lproj/*)
+        git restore Blockzilla.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved
```

Comment:
@razvanlitianu I have done a few tests and that should work. Worth giving it a try 🤞